problem_builder/mentoring.py

```diff
@@ -963,6 +967,11 @@ class MentoringWithExplicitStepsBlock(BaseMentoringBlock, StudioContainerWithNes
 
     editable_fields = ('display_name', 'max_attempts', 'extended_feedback', 'weight')
 
+    def build_user_state_data(self, context=None):
+        user_state_data = super(MentoringWithExplicitStepsBlock, self).build_user_state_data()
+        user_state_data['active_step'] = self.active_step_safe
```

Comment:
Do we need `active_step` in `USER_STATE_FIELDS` if we're setting it here explicitly anyways? Isn't that wasting cycles to set it then overwrite it?